SSA/Experimental/ErasedContext.lean

```diff
@@ -191,26 +191,18 @@ theorem Valuation.snoc_toSnoc {Γ : Ctxt} {t t' : Ty} (s : Γ.Valuation) (x : t.
 /- ## VarSet -/
 
 /-- A `Ty`-indexed family of sets of variables in context `Γ` -/
-def VarSet (Γ : Ctxt) : Type := 
-  (t' : Ty) → Finset (Γ.Var t')
+abbrev VarSet (Γ : Ctxt) : Type := 
+  Finset (Σ t, Γ.Var t)
 
 namespace VarSet
 
 /-- A `VarSet` with exactly one variable `v` -/
 @[simp]
 def ofVar {Γ : Ctxt} (v : Γ.Var t) : VarSet Γ :=
-  fun t' => if ty_eq : t = t' then {ty_eq ▸ v} else {}
-
-@[simp]
-instance : EmptyCollection (VarSet Γ) := ⟨fun _ => ∅⟩
```

Comment:
why do we want to remove these instances?

Reply:
Now that `VarSet` is just an abbrev for a `Finset _`, these instances are found directly as the `Finset` instances